src/main/java/org/kuali/kfs/module/cam/document/service/impl/GlLineServiceImpl.java

```diff
@@ -408,8 +414,20 @@ protected boolean doesGeneralLedgerEntryMatchAssetAccountingDetails( GeneralLedg
         }
         
         //check if GL Debit matches acct line type target and Credit matches Source
-		if ((StringUtils.equals(entry.getTransactionDebitCreditCode(), KFSConstants.GL_DEBIT_CODE) && StringUtils.equals(accountingDetails.getFinancialDocumentLineTypeCode(), KFSConstants.TARGET_ACCT_LINE_TYPE_CODE))
-				|| (StringUtils.equals(entry.getTransactionDebitCreditCode(), KFSConstants.GL_CREDIT_CODE) && StringUtils.equals(accountingDetails.getFinancialDocumentLineTypeCode(), KFSConstants.SOURCE_ACCT_LINE_TYPE_CODE))) {
+		boolean isGlEntryDebit = StringUtils.equals(entry.getTransactionDebitCreditCode(), KFSConstants.GL_DEBIT_CODE);
+        boolean isGlEntryCredit = StringUtils.equals(entry.getTransactionDebitCreditCode(), KFSConstants.GL_CREDIT_CODE);
+        boolean isAccountingDetailTarget = StringUtils.equals(accountingDetails.getFinancialDocumentLineTypeCode(), KFSConstants.TARGET_ACCT_LINE_TYPE_CODE);
+        boolean isAccountingDetailSource = StringUtils.equals(accountingDetails.getFinancialDocumentLineTypeCode(), KFSConstants.SOURCE_ACCT_LINE_TYPE_CODE);
+        boolean isGeneralErrorCorrection = StringUtils.equalsIgnoreCase(entry.getFinancialDocumentTypeCode(), KFSConstants.FinancialDocumentTypeCodes.GENERAL_ERROR_CORRECTION);
+        if (LOG.isDebugEnabled()) {
```

Comment:
I think all there logging statements could be removed now that development is done. They clutter the code more than they help at this point.

Reply:
How about moving these logs to a separate function, that will keep this function cleaner, I think this log statement will be helpful

Reply:
that makes sense, thank you